Ex1/week5_1.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+import os
+import random
+
+# That's the way
+
+
+def find_files_in_directory_startwith_deep(path):
```

Comment:
rename path variable name to directory_path, then you can rename your function to something like  
find_files_with_deep_prefix(directory_path) , I think its better, your choice